public/app/config/graaspAlgorithms.js

```diff
@@ -143,6 +143,40 @@ const GRAASP_ALGORITHMS = [
       },
     ],
   },
+  {
+    id: 'shuffle-fields',
+    name: 'Shuffle fields',
+    description: 'Shuffle selected array fields from a dataset',
```

Comment:
'Given an array of objects, this algorithm randomly shuffles the values in selected keys between the array's objects.'

A bit of a mouthful, but...!